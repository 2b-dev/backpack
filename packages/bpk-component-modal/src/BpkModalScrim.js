import React from 'react';
import { TransitionInitialMount } from 'bpk-react-utils';

import './bpk-modal.scss';

const BpkModalScrim = () => {
  const className = 'bpk-modal-scrim';

  return (
    <TransitionInitialMount classNamePrefix={className} transitionTimeout={200}>
      <div className={className} />
    </TransitionInitialMount>
  );
};

export default BpkModalScrim;
