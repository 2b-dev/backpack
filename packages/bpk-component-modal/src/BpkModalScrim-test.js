import React from 'react';
import renderer from 'react-test-renderer';
import BpkModalScrim from './BpkModalScrim';

describe('BpkModalScrim', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <BpkModalScrim />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
