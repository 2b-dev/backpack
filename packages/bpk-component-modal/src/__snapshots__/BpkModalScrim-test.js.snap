exports[`BpkModalScrim should render correctly 1`] = `
<div
  className="bpk-modal-scrim" />
`;
